tests/builtin_msg_field_types_generators.py

```diff
@@ -75,3 +76,9 @@ def test_float64_generates_expected_min_value_as_default(generated_value):
     """Verify default min. generated value for Float64."""
     assert generated_value >= builtin_msg_field_types.FLOAT64_MIN_VALUE
     assert generated_value <= builtin_msg_field_types.FLOAT64_MAX_VALUE
+
+@given(builtin_msg_field_types.date())
+def test_date_generates_expected_min_value_as_default(generated_value):
```

Comment:
This seems to be a copy & paste issue as well. (I need to fix that as well.) What about `test_date_generates_expected_value_range` or something similar?